### ISSUE TITLE
ROBOT report --print X fails if less than X errors

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#544]
```
	$(ROBOT) -vvv report -i $< --fail-on $(REPORT_FAIL_ON) --print 5 -o $@
```

Will fail if there are less than 5 errors; Example output:

```
Violations: 2
-----------------
ERROR:      0
WARN:       1
INFO:       1

2019-08-01 15:40:44,142 DEBUG org.obolibrary.robot.ReportOperation - Writing report to: reports/mo-edit.owl-obo-report.tsv

First 5 violations:
WARN	missing_definition	MO:00000000	IAO:0000115		
INFO	missing_superclass	MO:00000000	rdfs:subClassOf		
2019-08-01 15:40:44,148 DEBUG org.obolibrary.robot.ExceptionHelper - Malformed exception message: 3
3
java.lang.ArrayIndexOutOfBoundsException: 3
	at org.obolibrary.robot.ReportOperation.printNViolations(ReportOperation.java:632)
	at org.obolibrary.robot.ReportOperation.report(ReportOperation.java:331)
	at org.obolibrary.robot.ReportCommand.execute(ReportCommand.java:118)
	at org.obolibrary.robot.CommandManager.executeCommand(CommandManager.java:248)
	at org.obolibrary.robot.CommandManager.execute(CommandManager.java:192)
	at org.obolibrary.robot.CommandManager.main(CommandManager.java:139)
	at org.obolibrary.robot.CommandLineInterface.main(CommandLineInterface.java:56)
make: *** [mo.Makefile:46: reports/mo-edit.owl-obo-report.tsv] Error 1
```

**Comment by rctauber** [comment]
Thanks for catching this. I just made a PR - easy fix 🙂

**Comment by matentzn** [comment]
Thanks!

**Comment by rctauber** [comment]
Resolved by #546
